libs/agno/agno/models/base.py

```diff
@@ -1028,11 +1028,13 @@ def _populate_stream_data_and_assistant_message(
         if model_response_delta.images:
             if stream_data.response_image is None:
                 stream_data.response_image = model_response_delta.images[-1]
+            should_yield = True
```

Comment:
Makes sense to me, but let's quickly check this didn't break the other test_multimodal batteries? the ones for the other models basically